state/executor.go

```diff
@@ -238,7 +238,8 @@ func (t *Transition) Write(txn *types.Transaction) error {
 	signer := crypto.NewSigner(t.config, uint64(t.r.config.ChainID))
 
 	var err error
-	if txn.From == emptyFrom {
+	// TODO: remove knowledge of TxTypeState
+	if txn.Type != types.TxTypeState && txn.From == emptyFrom {
```

Comment:
Leftover TODO.

What does it mean?

Reply:
Fixed eddbd35203739b6fe17c9e13481e676cf9fa36d4